.github/pull_request_template.md

```diff
@@ -6,9 +8,14 @@ Changes proposed in this pull request:
 
 After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.
 
-For nontrivial changes, we will conduct both a design review and a code review. Please tag a core team member for code review, or leave blank and we will find someone for you!
+For nontrivial changes, we will conduct both a design review and a code review. 
+
+Design Review will include ensuring that the PR is consistent with our relationship to the open source community and partner developers, with our software architecture and abstractions, and with our users' needs and expectations. We try to effectively conduct design review as early as possible, often well before a PR. Please find us via issues, discuss, or slack to engage, and then succinctly describe those interactions in the "design review" section below.
```

Comment:
I bet we can boil this down so it's less intimidating. We don't need to block on that, though.